### ISSUE TITLE
Compatibility with original Raspberry images?

### ISSUE THREAD

**Issue by dedekcz** [bigtreetech/BTT-Pi#1]
Hello,

Is this board compatible with official Raspberry image?

THX,RH

**Comment by Tompudex** [comment]
No, it isn't. the iso that can be used separately for cb modules is also suitable for this

**Comment by Nbrooks639** [comment]
i am having issues with reconnecting to mainsail after a simple save and restart from any cfg file. i have to completely shutdown and reboot
